examples/kotlinExample/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.0.0'
+    ext.kotlin_version = '1.0.1'
```

Comment:
This PR is branched off from [`feature/objectid`](https://github.com/realm/realm-java/tree/feature/objectid) and @zaki50 work was merge from master to avoid conflict.